docs/building-images.md

```diff
@@ -56,6 +58,11 @@ operatingSystem:
       - serviceX
 ```
 
+* `installDevice` - Optional; only for ISO images - specifies the disk that should be used as the install
+  device. This needs to be block special, and will default to automatically wipe any data found on the disk.
+* `unattended` - Optional; only for ISO images - forces GRUB override to automatically install the operating
```

Comment:
Aside from these two being ISO only, do we want to validate anything else? Should we raise a GitHub issue to add the validation steps or you prefer to include those with the tests you've mentioned you've planned?

Reply:
I've just added them... if the user now selects "raw" then neither of these options are allowed.